isis/tests/Fixtures.cpp

```diff
@@ -224,7 +224,7 @@ namespace Isis {
     projCubeLabel >> projLabel;
 
     testCube = new Cube();
-    testCube->fromIsd(tempDir.path() + "default.cub", label, isd, "rw");
+    testCube->fromIsd(tempDir.path() + "/default.cub", label, isd, "rw");
```

Comment:
This basically caused most of them, sometimes the "/" was added but on Jenkins it wasn't? Bizarre. Might need to start using a Path library or add path concatenations to the FileName class.